docs/chat-configuration/ui-customization/incoming-message/carousel.md

```diff
@@ -51,35 +53,40 @@ Available configuration proprties can be viewed on `CarouselItemsUIAdapter`
 
 
 ### Info section
-Info section contains a title and a sub-title, which are configured with minimum line number of 1 and 2 lines respectfully.   
-Minimun height for this section will be calulated according to those values.
+The info section contains a title and a sub-title. The numbr of lines for each is configurable with the following methods:
+-  `setInfoTitleMinLines` - minumum is one line
+- `setInfoSubTitleMinLines` - minimum are two lines
+
 The sub-title height is flexible and may stretch to the maximum available item height.
 
-Minimum line numbers can be configured with `setInfoTitleMinLines` and `setInfoSubTitleMinLines` methods.
+The Minimun height for this section will be calulated according to the definenumber of lines described above.  
+
 
 ![]({{'/assets/images/carousel-info.png' | relative_url}})
 {: .image-40}
 
-> Checkout other configurations for the info section on `CarouselItemsUIAdapter` prefixed with **setInfo...**.
+> Additional info section configurationscan be found in the  `CarouselItemsUIAdapter` prefixed with **setInfo...**.
+
+##COMMENT - CAN WE PROVIDE THE LIST HERE?
```

Comment:
why? they can see it in the code.
I we'll start listing methods, we need to do it for other components as well

Reply:
OK